linux/guest_customization/check_gosc_support_status.yml

```diff
@@ -33,6 +33,26 @@
         - vcenter_version is version ('7.0.1', '<')
         - not enable_cloudinit_gosc | bool
 
+    - block:
+        - include_tasks: ../utils/get_installed_package_info.yml
+          vars:
+            package_name: "open-vm-tools"
+
+        - name: "Set fact of GOSC support status to False for Photon OS with not bundled open-vm-tools"
+          set_fact:
+            gosc_is_supported: False
+          when:
+            - not enable_cloudinit_gosc | bool
+            - package_info is defined
+            - package_info.Release is defined
+            - "'ph' not in package_info.Release | string"
+
+        - include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "GOSC is not supported for {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_ver }} with open-vm-tools {{ vmtools_version }}."
```

Comment:
If this only apply to perl GOSC, then I think this 'skip_msg' can be more specific. And `when: not enable_cloudinit_gosc | bool` can be moved to below with `when: guest_os_ansible_distribution == 'VMware Photon OS'`?

Reply:
Yes, cloud-init GOSC works with not bundled open-vm-tools. The tasks have been updated. Thanks!